prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -115,8 +115,7 @@ fi
 # Workflows disabled in async mode
 if [ $CTFINPUT == 0 ]; then
   WORKFLOW+="o2-phos-reco-workflow $ARGS_ALL --input-type raw --output-type cells $DISABLE_MC  | "
```

Comment:
Also phos now equipped with readers/writers, could you add `` --disable-root-input --disable-root-output `` here?